app.js

```diff
@@ -1,5 +1,51 @@
-// import functions and grab DOM elements
+import { getRandomThrow, doesUserWin} from './get-random-throw.js';
```

Comment:
looks like you're getting an error from github, just need to put the space after n in 'doesUserWin'